polling_stations/apps/data_importers/management/commands/teardown.py

```diff
@@ -76,10 +80,13 @@ def handle(self, *args, **kwargs):
             print("Deleting ALL data...")
             PollingDistrict.objects.all().delete()
             PollingStation.objects.all().delete()
-            ResidentialAddress.objects.all().delete()
+
             # use raw SQL so we don't have to loop over every single record one-by-one
             cursor = connection.cursor()
             cursor.execute(
-                "UPDATE data_collection_dataquality SET report='', num_addresses=0, num_districts=0, num_stations=0"
+                "UPDATE addressbase_uprntocouncil SET polling_station_id='' WHERE polling_station_id != ''"
```

Comment:
In django's ORM, if you declare a string column, the default is that the empty value is `''` not `NULL`. So `CharField()` implies `CharField(null=False)`, essentially. Because `polling_station_id` is just a `CharField()` we should use empty string as the "blank" value because that's what the ORM will do/expect.

https://github.com/GeoWill/UK-Polling-Stations/blob/c3a0e54a31257316292f876e73c7a6d4df5764ed/polling_stations/apps/addressbase/models.py#L103

Reply:
Just re-reading this, is there actually any benefit to making this a raw SQL statement rather than using the ORM to do `UprnToCouncil.objects.filter(polling_station_id='').update(polling_station_id="")` (I've not checked)

Reply:
raw sql is slightly faster over ~1million rows. 
- raw sql ~19 seconds:  `connection.cursor().execute("UPDATE addressbase_uprntocouncil SET polling_station_id='' WHERE polling_station_id != ''")`
- ORM ~20seconds: `UprnToCouncil.objects.exclude(polling_station_id='').update(polling_station_id='')`